### DISCUSSION TITLE
rmview displays color pdfs and small edit to readme

### DISCUSSION THREAD

**Discussion by TedwardErker** [bordaigorl/rmview#125] [Ideas]
Hi,

I'm not really sure where to say this, so I'm starting this discussion.  First, this is an awesome tool. Thanks for creating it!

I have two small suggestions to add to the readme that I think will help future users appreciate what rmview can do and also help them get set up.

1) I think it would be helpful for users to know that a pdf that has color won't show the color on the remarkable tablet, but will on rmview.  Maybe this is obvious to some people, but is wasn't to me and really increases the utility of rmview for me.  

2) In the manual installation section of the readme, it says "Then you can run the program with python -m rmview."  It would be helpful if you edited this to say "Then you can run the program with python -m rmview _from the src directory_".  This is mentioned in another discussion which is how I figured it out, but it wasn't clear for me in the readme.

thanks again!

**Comment by bordaigorl** [comment]
Thanks for the feedback!
(I don't get this new discussion thing on GitHub, an issue would have been fine for this)
I updated the readme according to your suggestions.
Let me also add: there is a Wiki which is freely editable, feel free to add info to it!

**Comment by toast-com** [comment]
Hi, 

Thanks for this very useful software.

TedwardErker mentioned that original pdf colors are preserved in rmview. This is not what I observe when I run it with "Screen Share". Pdf files remain in greyscale. Is there any specific parameters to activate in the configuration file?

PC running Ubuntu 20.04.3 LTS
RM2 v2.12.1.527

Thanks

**Reply by bordaigorl** [reply]
...It should work out of the box. Make sure you have an up-to-date version of rmview though

**Reply by toast-com** [reply]
Sorry, I didn't see there was a bug fix related to color outputs in v3.1. I was running v3.0. It works fine now.
Thanks a lot
